content/blog/e2e-encryption/index.md

```diff
@@ -0,0 +1,65 @@
+---
+path: '/e2e-encryption'
+title: 'Adding End-to-End Encryption for Proxied Data'
+date: 2021-9-21T06:00:00.000+00:00
+featured_image: e2e-hero.png
+categories: ['Pixie Team Blogs']
+authors: ['Vihang Mehta']
+emails: ['vihang@pixielabs.ai']
+featured: true
+---
+
+End-to-end encryption has become increasingly popular as consumers demand that any data they send - a file, email, or text message - is not decipherable by any unauthorized recipients. This consumer trend is evident in the recent surge in popularity of Signal, an encrypted instant messaging service. 
+
+In this post, we’ll cover what end-to-end encryption is and walk you through how we implemented it in our system.
```

Comment:
could consider saying that we link the commits at the end of the post if you want to see the code

Reply:
happy to reiterate if you feel strongly but otherwise I'm trying to keep the total length manageable